src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/BookmarksModal.tsx

```diff
@@ -55,16 +57,22 @@ export const PreviouslyViewedLinks = () => {
           focus: buildFocusIdForUrl(previouslyViewedObject.object_id)
         });
 
+        const isFocusCurrentlyActive =
+          focusObjectId === previouslyViewedObject.object_id;
+
+        const onClick = isFocusCurrentlyActive
+          ? (event: React.MouseEvent<HTMLAnchorElement>) => {
+              changeFocusObject(focusObjectId);
+              event.preventDefault();
```

Comment:
I've usually seen `event.preventDefault()` put in the top of the function. But think this should be okay too.

Reply:
I'd also put it in the beginning of the function, out of habit. Don't think it really matters one way or the other though.